web/controllers/stream_controller.ex

```diff
@@ -50,6 +50,39 @@ defmodule Streamr.StreamController do
     render conn, "show.json-api", data: stream
   end
 
+  def delete(conn, %{"id" => id}) do
+    stream = Repo.get!(Stream, id)
+
+    case Repo.delete(stream) do
+    {:ok, stream} ->
+      conn
+      |> put_status(204)
+      |> render("show.json-api", data: Repo.preload(stream, :user))
```

Comment:
For some reason, I tried doing this and even looked a bit into it, however, whenever I would do `conn |> send_resp(204, "")` it required it to send JSON back but even when I would change it to `conn |> put_resp_header("content-type", "application/json") |> send_resp(204, "")`, it did not like that.

However, I just changed it to doing `put_status(204) |> json("")`